Src/ReadBatteryVoltage.c

```diff
@@ -0,0 +1,37 @@
+#include "stm32f4xx.h"
+#include "stm32f4xx_hal_conf.h"
+#include "cmsis_os.h"
+
+#include "ReadBatteryVoltage.h"
+
+#include "Data.h"
+
+static int READ_BATTERY_VOLTAGE_PERIOD = 250;
```

Comment:
This could probably be increased to 1000 or more. I don't think we need to be sampling battery voltage that often.

Reply:
Increasing to 1000.